gramps/plugins/importer/importxml.py

```diff
@@ -279,21 +278,21 @@ def info_text(self):
         Construct an info message from the data in the class.
         """
         key2string = {
-            PERSON_KEY      : _('  People: %d\n'),
-            FAMILY_KEY      : _('  Families: %d\n'),
-            SOURCE_KEY      : _('  Sources: %d\n'),
-            EVENT_KEY       : _('  Events: %d\n'),
-            MEDIA_KEY       : _('  Media Objects: %d\n'),
-            PLACE_KEY       : _('  Places: %d\n'),
-            REPOSITORY_KEY  : _('  Repositories: %d\n'),
-            NOTE_KEY        : _('  Notes: %d\n'),
-            TAG_KEY         : _('  Tags: %d\n'),
-            CITATION_KEY    : _('  Citations: %d\n'),
+            PERSON_KEY      : _('\tPeople{colon}\t%d\n').format(colon=COLON),
```

Comment:
Using tabs in the middle of a string like this doesn't really work.

Reply:
In the picture above you can see work this tabs.
Using \t I can align digit after text.

Reply:
Translator add needed '\t' in .po file for align word in his language.